hpat/tests/test_series.py

```diff
@@ -122,6 +122,55 @@ def gen_strlist(size, nchars=8):
     return [''.join(chars) for chars in generated_chars]
 
 
+def restore_series_argsort(series, my_result):
```

Comment:
Please, add doc-string to clarify what this func does

Reply:
Fixed